docs_versioned_docs/version-ros2humble/ros/config/yaml/mounts.mdx

```diff
@@ -0,0 +1,505 @@
+---
+title: Mounts
+sidebar_label: Mounts
+sidebar_position: 6
+toc_min_heading_level: 2
+toc_max_heading_level: 4
+---
+Most sensors can use similar mounting structures. Therefore, we want to keep mounts separate from sensors, such that users could attach their own sensors to existing mounts.
+
+## Riser
+A mounting plate with the defined number of rows and columns of the PACS™ **80mm x 80mm** grid.
+> :warning: At this time, you will **need to use the PACS™ top plate**, only available on the Husky, to add these riser mounts. See the [**platform**](./platform.mdx) section for more information.
```

Comment:
Is husky PACS plate is now available on all platforms?

Reply:
It is available on all platforms, but I don't know how you'd add it realistically on a Jackal or Warthog. The issue is the inheritance of the grid naming scheme. We need to update it so that if the parent link doesn't have the appropriate `{letter_column}{number_row}` scheme that the added riser just starts its own naming scheme.

Reply:
But the top plate can be added without the parent being PACS. Risers would just have to go onto the top plate.